runtime/neumann/src/lib.rs

```diff
@@ -165,10 +165,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("neumann"),
 	impl_name: create_runtime_str!("neumann"),
 	authoring_version: 1,
-	spec_version: 278,
+	spec_version: 280,
```

Comment:
why not `278` for neuman?

Reply:
whoops, not 280. should be 279. 278 is already in production. that was in the 1.2.8 version bump